docs/interfaces/datastore-service.txt

```diff
@@ -84,14 +84,28 @@ Interface CreateEvent {
     }
 }
 
-// Note: For deleting keys, they must be set to `None`. These keys
-// will be removed from the model.
+// Note: For deleting keys, they must be set to `None`. These keys will be removed from
+// the model.
+// list_fields can be used to partially update list fields: the values in `add` will be
+// appended to the given field, the values in `remove` will be removed from the field.
+// The case of double adding or removing something invalid will be silently ignored.
```

Comment:
"ignored" meaning it is done and not logged/noticed, or just not done? This needs to be more precise